partials/footer.hbs

```diff
@@ -3,27 +3,96 @@ This footer template is shared across all the pages.
 --}}
 
 <footer class="m-footer">
-  <div class="m-footer__content">
-    <p class="m-footer-copyright">
+  <div class="footer-content row">
+    <div class="four-columns footer-columns">
+      <div class="footer-brand-div text-center">
+        <h2>University of California, Berkeley IEEE Student Branch</h2>
+        <p>251 CoryHall, University of California, Berkeley</p>
+      </div>
+    </div>
+
+    <div class="four-columns footer-columns">
+      <div class="text-center">
+        <h3 class="footer-heading">Links</h3>
+        <p>
+          <a href="#" class="footer-link">FAQ</a><br />
+          <a href="#" class="footer-link">Check-In</a><br />
+          <a href="#" class="footer-link">About Us</a><br />
+          <a href="#" class="footer-link">Sponsors</a><br />
+          <a href="#" class="footer-link">Resources</a>
+        </p>
+      </div>
+    </div>
+
+    <div class="four-columns footer-columns text-center">
+      <h3 class="footer-heading">For Officers</h3>
+      <p>
+          <a href="#" class="footer-link">Blog</a><br />
+          <a href="#" class="footer-link">Check In</a><br />
+          <a href="#" class="footer-link">Execs and Officers</a><br />
```

Comment:
Todo

Reply:
Oh yeah, I wasn't sure how to add those links actually, because I don't think I can use an absolute web address there right?

Reply:
Oh! By using a forward slash, you reference the link from the website:
For instance, if your website was `https://site.com/`
`href="/tothispage"`
is the same as
`href="https://site.com/tothispage"`

You just have to add in `href="/blog/<add the page link here>"`

Reply:
Would be careful about prepending `/blog`, since this will eventually be in production.

I think this is done better through the Ghost interface. There is a "secondary navigation" option that Ghost provides, which could be repurposed for footer links. I found [some documentation here](https://ghost.org/docs/api/v3/handlebars-themes/helpers/navigation/), but from what I can tell this would be similar to how the top header gets links from Ghost.

Reply:
Whoops! Kevin is totally correct here. It should be:
`href="{{ @site.url }}/<route to your page>"`

Reply:
Oh yeah I noticed that while actually fixing this. Thanks!